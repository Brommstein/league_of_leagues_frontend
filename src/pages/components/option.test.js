import React from 'react';
import Option from './option';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

describe('Option Component', () => {
    const props = {
        allUsers: {
            userid: 56,
            leaguename: 'ComponentTest'
        }
    }
    it('component', () => {
        const optionComponent = shallow(<Option {...props} />);
        expect(optionComponent).toMatchSnapshot();
    });
});