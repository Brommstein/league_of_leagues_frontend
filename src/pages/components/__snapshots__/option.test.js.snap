// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Option Component component 1`] = `ShallowWrapper {}`;
